combat-trainer.lic

```diff
@@ -428,10 +428,12 @@ class LootProcess
       if @necro_heal && !game_state.necro_casting?
         game_state.wounds = check_health
         echo "Severity to Wounds: #{game_state.wounds}" if $debug_mode_ct
-        unless game_state.wounds.empty?
-          do_necro_ritual(mob_noun, 'consume', game_state)
-          return false
-        end
+		if @heal_threshold <= game_state.wounds.keys.max
```

Comment:
any chance you can set your indents to 2 spaces as per the Ruby style?

Reply:
Rubocop will fix this automagically for you if you're interested: https://github.com/bbatsov/rubocop

Running `rubocop -a` in the scripts directory will autocorrect all script files

Reply:
Nifty, I will use this next time.